src/core/components/accordion/index.tsx

```diff
@@ -25,7 +25,7 @@ const visuallyHidden = css`
 	${_visuallyHidden}
 `;
 
-interface AccordionProps extends Props {
+export interface AccordionProps extends Props {
```

Comment:
Do we need to export these Props interfaces in order for consuming projects to understand what props are needed for each component?

Reply:
i think so yeah

Reply:
typescript was complaining, i guess because the rollup build was combining all the exports in one place?

Reply:
Right. I guess if TypeScript is complaining, it should prevent us from accidentally removing that `export` keyword in the future. Just thinking that this looks like a Chesterton's Fence in waiting!